tests/unit/test_firehose.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from localstack.services.firehose import firehose_api
+
+TEST_STREAM_NAME = 'test_stream'
+TEST_TAG_1 = {'Key': 'MyTag', 'Value': 'TestValue'}
+TEST_TAG_2 = {'Key': 'AnotherTag', 'Value': 'AnotherValue'}
+TEST_TAGS = [TEST_TAG_1, TEST_TAG_2]
+
+
+class FirehoseApiTest(unittest.TestCase):
+
+    def setUp(self):
+        firehose_api.create_stream(TEST_STREAM_NAME, tags=TEST_TAGS)
```

Comment:
nitpick: perhaps worthwhile adding a `tearDown(..)` method as well, to delete the test stream.

Reply:
Agreed, done.